lighthouse-core/test/audits/metrics/speed-index-test.js

```diff
@@ -28,19 +28,4 @@ describe('Performance: speed-index audit', () => {
       assert.equal(result.rawValue, 605);
     });
   }, 10000);
-
-  it('scores speed index of 845 as 100', () => {
```

Comment:
the test above already tests that a score of 605 yields a passing score. This seemed redundant (and difficult to fake)